Documentation/Signoffs/Sensor_Signoff.md

```diff
@@ -0,0 +1,86 @@
+
+
+### Rotary Torque Sensor Wiring Schedule:
+ ![image](https://github.com/user-attachments/assets/cc51497b-23e1-4309-8449-b6eeedec3bc1)
+ ![image](https://github.com/user-attachments/assets/f9c313e3-b487-4814-b57c-716658497eb2)
+
+The image above shows the wiring schedule for the torque sensor cable. The connections are to the DAQ (NI USB_6215) and a USB to RS485 converter. This allows the user to program the sensor's parameters and provides a redundant data display to ensure the DAQ output matches the sensor's output. The ninth pin is connected to the cable shield, which will be attached to terminal 11 on the DAQ to eliminate any chance of noise generated onto the signal from electromagnetic interference.
+
+
+
+### Rotary Torque Sensor Installation Dimensions:
+
+![image](https://github.com/user-attachments/assets/04b262e3-b051-4b7a-8d39-33ea47275d50)
+
+The image above shows the required dimensions in millimeters for mounting the sensor into the dynamometer.
+
+## Analysis
+### Rotary Torque Sensor:
+The torque sensor is a transducer that converts mechanical torque into an electric output signal. This is achieved by attaching strain gauges to the shaft. As the shaft's torsion increases, the strain gauges' value proportionally changes, and the change in voltage or resistance measured is used to calculate the shaft's induced torque. The bike’s maximum torque from the 3500 W drive motor is 25.94 lb-ft. With a 20% safety factor, the system must then be able to measure up to 31.13 lb-ft. The selected sensor can support up to 100 Nm, which converts to 73.756 lb-ft. 
+
+The sensor must also support the drum’s maximum RPM driven from the bike's drive wheel, which is 1,288 RPM. The selected sensor is rated for up to 10,000 RPM and will support up to 15,000 RPM. The sensor will be powered by a 24-volt power supply and output a 0-5 volt signal. The 
+
+### RPM Sensor:
+The rpm sensor is internally incorporated into the selected component. The RPM is measured with an encoder that reads the rotations per minute of the shaft passing through the device. This value at the sensor will be higher than the actual value of the bike’s wheel, but it can be converted after calculating the gearing ratio and implementing that into the DAQ software. The drum will be 12 inches in diameter, and the shaft on the sensor is 14.5mm or 0.571 inches in diameter. The diameter of the drum divided by the shaft equals a 21.016 gear ratio. Therefore, when the dynamometer rotates at the maximum RPM of 1,288, the sensor’s shaft speed will measure 27,068 RPM.
```

Comment:
I am not understanding something here. The gear ratio is the bike wheel to the drum, not the drum to the sensor shaft. The sensor shaft is coupled directly to the drum right?

Reply:
Correct, my apologies for the confusion. The gear ratio was re-calculated using the bike's rear tire to the drum since the sensor is directly coupled to the shaft from the drum. I clarified this in my revision.